matrix-sdk/src/main/java/org/matrix/androidsdk/core/EventDisplay.java

```diff
@@ -297,7 +297,17 @@ public CharSequence getTextualDisplay(Integer displayNameColor, Event event, Roo
                     displayName = redactedInfo;
                 }
 
-                text = mContext.getString(R.string.notice_room_third_party_invite, userDisplayName, displayName);
+                if (displayName != null) {
+                    text = mContext.getString(R.string.notice_room_third_party_invite, userDisplayName, displayName);
+                } else {
+                    // Consider the invite has been revoked
+                    JsonObject prevContent = event.getPrevContentAsJsonObject();
+                    if (prevContent != null) {
+                        text = mContext.getString(R.string.notice_room_third_party_revoked_invite, userDisplayName, prevContent.get("display_name"));
```

Comment:
Handle the case where the display name is null?

Reply:
no this is not relevant...